src/test/java/org/folio/edge/sip2/handlers/PatronStatusHandlerTests.java

```diff
@@ -98,7 +99,67 @@ void canExecutePatronStatus(Vertx vertx,
         })));
   }
 
+  @Test
+  void cantExecutePatronStatus(Vertx vertx,
+                              VertxTestContext testContext,
+                              @Mock PatronRepository mockPatronRepository) {
```

Comment:
This style of indents is different from the other tests. Would it be good to make them all the same? I like this style which intellij will help with, but this one needs to come forward 1 space.